tests/baselines/reference/tsxStatelessFunctionComponentOverload4.errors.txt

```diff
@@ -49,27 +39,25 @@ tests/cases/conformance/jsx/file.tsx(36,29): error TS2322: Type '{ children: str
     
     // Error
     const c0 = <OneThing extraProp />;  // extra property;
-                         ~~~~~~~~~

```

Comment:
Should the span still be on extraProp here since it's the failing prop?

Reply:
+1

Reply:
I mean, before it was on the attributes as a whole (it just happens there's only one attribute in this example, so it happens to look the same).

Reply:
Also, you may have misread the error: it's not the failing prop (yet - it would be an excess prop assuming it was otherwise compatible) - this example is missing both `yy` and `yy1`.